webui/ocw/lib/vault.py

```diff
@@ -75,7 +81,12 @@ def getCredentials(self):
 
     def getData(self, name=None):
         if self.auth_json is None:
-            self.auth_json = self.getCredentials()
+            if self.use_file_cache:
+                self.auth_json = self.loadFileCache()
```

Comment:
two things possible - or you name your patch wrong or you should call `self.auth_json = self.getCredentials()` unconditional to `use_file_cache`  . because what I see here is not "caching the response from server" , but replacing it

Reply:
It's only set if it isn't set previously...